modules/emqtt_bench/main.tf

```diff
@@ -8,14 +18,14 @@ module "emqtt_bench_ec2" {
   sg_ids            = var.sg_ids
   s3_bucket_name    = var.s3_bucket_name
   iam_profile       = var.iam_profile
-  instance_name     = "${var.namespace}-emqtt-bench"
```

Comment:
Not exactly.

ec2 module overrides this tag: https://github.com/emqx/tf-emqx-performance-test/blob/master/modules/ec2/main.tf#L38

We do not need namespace in instance name because now we create unique hosted zone per benchmark id which already includes namespace: https://github.com/emqx/tf-emqx-performance-test/pull/7/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR92

Before this we used static hosted zone.